cypress/e2e/allEmployees/addEmployee.spec.cy.js

```diff
@@ -0,0 +1,65 @@
+import * as dataUtils from '../../fixtures/fakerData.js'
+import employeeDatas from '../../fixtures/addEmployeeData.json'
+
+describe('addEmployee', () => {
+
+    let loginTestData, employeeData;
+    employeeData = dataUtils.randomData()
+    const id = Math.floor(Math.random() * 1000) + 100;
+
+    before(() => {
+        cy.fixture('loginData').then((user) => {
+            loginTestData = user;
+        })
+    })
+
+    it.skip('addEmployeeAsAdmin', () => {
```

Comment:
actually I skipped this it block  (addEmployeeAsAdmin) because The second it block (loginWithCreatedEmployeeAsAdmin) has same flow . Now I removed skip.